docs/text-style-props.md

```diff
@@ -777,6 +777,8 @@ export default App;
 | ------ |
 | string |
 
+Note: The generic font families `system-ui`, `ui-sans-serif`, `ui-serif`, `ui-monospace`, and `ui-rounded` are supported on iOS (since version 0.78).
```

Comment:
Hey @cxa, thanks for the update! 👍 Let's tweak the wording a bit:

```suggestion
The generic font families `system-ui`, `ui-sans-serif`, `ui-serif`, `ui-monospace`, and `ui-rounded` are supported on iOS.
```

Can you also add this note to the versioned docs for 0.78 and above (instead of mentioning the version in text directly)? Versioned files are located here:
* https://github.com/facebook/react-native-website/tree/main/website/versioned_docs